{"version":1,"defects":{"Tests\\Unit\\RouterTest::test_it_register_a_route":3,"Tests\\Unit\\RouterTest::test_it_register_a_get_route":3,"Tests\\Unit\\RouterTest::test_there_are_no_route_when_router_is_created":4,"Error":4,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #0":3,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #3":3,"Tests\\Unit\\RouterTest::test_it_resolve_a_route_from_a_clouser":4,"Tests\\Unit\\RouterTest::test_it_resolve_a_route":4,"Tests\\Unit\\Services\\InvoiceServiceTest::test_it_process_invoice":3,"Tests\\Unit\\Services\\InvoiceServiceTest::test_it_sends_receipt_email_when_process_method_is_call":3},"times":{"Tests\\Unit\\RouterTest::test_it_register_a_route":0.074,"Tests\\Unit\\RouterTest::test_it_register_a_get_route":0,"Tests\\Unit\\RouterTest::test_it_register_a_post_route":0,"Tests\\Unit\\RouterTest::test_there_are_no_route_when_router_is_created":0.003,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #0":0.013,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #1":0,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #2":0,"Tests\\Unit\\RouterTest::test_it_throw_route_not_found_excepiton with data set #3":0,"Error":0.121,"Tests\\Unit\\RouterTest::test_it_resolve_a_route_from_a_clouser":0,"Tests\\Unit\\RouterTest::test_it_resolve_a_route":0,"Tests\\Unit\\Services\\InvoiceServiceTest::test_it_process_invoice":0.033,"Tests\\Unit\\Services\\InvoiceServiceTest::test_it_sends_receipt_email_when_process_method_is_call":0.002}}